auth-server/oauth-scim-service/src/main/resources/templates/login.html

```diff
@@ -66,7 +66,7 @@
                     <div>
                       <div>
                         <div class="text-center login-logo">
-                          <img th:src="@{/images/google-logo.png}"
+                          <img th:src="@{/images/landingpage-logo.png}"
```

Comment:
Logo needs to be reverted to how it was provided originally - remove the superimposed text

Reply:
All image related changes will be addressed in release 2.0.3.
Issues #3181, #3182, #3183, #3184 and #3185, to update the images for all modules.